src/sentry/conf/server.py

```diff
@@ -1036,11 +1036,10 @@ def SOCIAL_AUTH_DEFAULT_USERNAME():
     "organizations:metrics": False,
     # Enable metric alert charts in email/slack
     "organizations:metric-alert-chartcuterie": False,
-    # Automatically extract metrics during ingestion.
-    #
-    # XXX(ja): DO NOT ENABLE UNTIL THIS NOTICE IS GONE. Relay experiences
-    # gradual slowdown when this is enabled for too many projects.
+    # Extract metrics for sessions during ingestion.
```

Comment:
The slowdown in Relay was resolved in https://github.com/getsentry/relay/pull/1065